app/views/user_mailer/batch_subscription_notification.text.erb

```diff
@@ -3,29 +3,44 @@
 <% @seen_summary = {} %>
 <% @creations.each_with_index do |creation, index| %>
 <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
-<%= this_work.pseuds.map{|p| p.byline}.to_sentence.html_safe %> posted a <% if creation.is_a?(Work) then %><%= this_work.backdate ? "backdated" : "new" %> work:<% else %>new chapter of <%= this_work.title %> (<%= this_work.word_count %> words):<% chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe %><% end %>
-<%= creation.is_a?(Work) ? work_url(this_work) : work_chapter_url(this_work, creation) %>
-
-<%= creation.is_a?(Work) ? this_work.title.html_safe : chapter_title %> (<%= creation.word_count%> words)<% unless @seen[this_work.id] %>
-by <%= this_work.pseuds.map{|p| text_pseud(p)}.to_sentence.html_safe %><% end %>
+<% pseud = this_work.pseuds.map{|p| p.name}.to_sentence %>
+<% byline = this_work.pseuds.map{|p| p.byline}.to_sentence %>
+<% title = this_work.title %>
+<% if creation.is_a?(Work) then
```

Comment:
This might be neater written as:

```
<%
  if creation.is_a?(Work)
    link = work_url(this_work)
    word_count = this_work.word_count
  else
    link = work_chapter_url(this_work, creation)
    chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe
  end
%>
```

I've also removed the `then` and the extra spaces between `link` and `=` 

Ideally, though, this should be in a helper method, not the view file.